src/CMakeLists.txt

```diff
@@ -64,18 +64,15 @@ endif()
 
 if (CMARK_SHARED)
   add_library(${LIBRARY} SHARED ${LIBRARY_SOURCES})
-  # Include minor version and patch level in soname for now.
   set_target_properties(${LIBRARY} PROPERTIES
+    MACOSX_RPATH TRUE
     OUTPUT_NAME "cmark"
+    # Avoid name clash between PROGRAM and LIBRARY pdb files.
+    PDB_NAME cmark_dll
```

Comment:
The property is only set on the library (this sets the property on the explicit target only).  This will have no behavioural difference, just is more compact and does all the updates in a single shot.

Reply:
Right, I misread the patch. So everything looks good to me.